.travis.yml

```diff
@@ -13,7 +13,7 @@ services:
   - docker
 
 env:
-  - TERRAFORM_VERSION=0.12.10 IMAGE_NAME=azure-network-module
+  - TERRAFORM_VERSION=0.12.20 IMAGE_NAME=azure-network-module

```

Comment:
I may miss some contexts but why it's 0.12.20 rather than 0.12.11?

Reply:
Because I use Terraform version 0.12.20, it correspond to the Terraform version. Since currently Terraform has updated to 0.12.24, I use terraform v0.12.20.